packaging/cmake/Config.cmake.in

```diff
@@ -0,0 +1,20 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+
+if(@WITH_ZLIB@)
```

Comment:
> what about SASL

SASL not found by `find_package`. As far as I understand it will be just full path to library (explicit location, no need to "find" anything) or something like `sasl2` (meaning standard path => `-lsasl2` to linker).

> LZ4?

I don't see the LZ4 in CMakeLists.txt (it's not very good probably). This config reflect configuration from CMakeLists.txt code.